docs/source/parallelism.mdx

```diff
@@ -269,8 +272,10 @@ Implementations:
 - [Megatron-LM](https://github.com/NVIDIA/Megatron-LM)
 - [Varuna](https://github.com/microsoft/varuna)
 - [SageMaker](https://arxiv.org/abs/2111.05972)
+- [OSLO](https://github.com/tunib-ai/oslo)
 
-🤗 Transformers status: not yet implemented
+🤗 Transformers status:
+- [OSLO](https://github.com/tunib-ai/oslo) is under discussion about integration with the Transformers.
```

Comment:
Until the integration happens, to the end user the current status before your change applies.

So let's not not confuse the user and update the status when the user can actually do something about it. 

Since you already added OSLO to the list of implementations should the user want to do something about it they can.

Same applies to the other few places where you propose to change status.

Reply:
Great.

Reply:
I reverted.